### ISSUE TITLE
green Lighting bug in cave Area

### ISSUE THREAD

**Issue by StijnStyno** [Interrupt/delverengine#277]
While playing, I came across a specific part of the cave area that was entirely covered in a green hue.
Maybe it has something to do with the lighting?
![cavebugreport](https://user-images.githubusercontent.com/92586070/152021794-60011942-82c8-4708-9561-d0a2f81ed3bb.png)
I was playing version 1.08 on windows 11.

**Comment by joshuaskelly** [comment]
Yeah. This is a side effect of the fog color. It's been around awhile, not 100% if it's intentional or not.
